java/vector/src/main/codegen/templates/DenseUnionVector.java

```diff
@@ -268,11 +270,11 @@ public long getDataBufferAddress() {
 
   @Override
   public long getValidityBufferAddress() {
-    return validityBuffer.memoryAddress();
+    return typeBuffer.memoryAddress();
```

Comment:
It seems `NullVector` throws `UnsupportedOperationException` in this case - should we be consistent with that?

Reply:
Ahh, yes. I copied UnionVector (which returns `typeBuffer`) but I didn't really like it. Fixed on both